src/LodeRunner.UI/src/components/ConfigPage/index.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import { A } from "hookrouter";
+import ConfigForm from "../ConfigForm";
+import "./styles.css";
+
+const ConfigPage = ({ configId }) => {
+  return (
+    <div className="config">
+      <div className="page-header">
+        <h1>
+          Load Test Config
+        </h1>
+        <A href="/" className="unset navigation">
```

Comment:
Should anchor be lowercase?

Reply:
A comes from: import { A } from "hookrouter";